pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -294,7 +300,32 @@ func (m *manager) GetTopologyHints(pod *v1.Pod, container *v1.Container) map[str
 	// Garbage collect any stranded resources before providing TopologyHints
 	m.removeStaleState()
 	// Delegate to active policy
-	return m.policy.GetTopologyHints(m.state, pod, container)
+	hint := m.policy.GetTopologyHints(m.state, pod, container)
+	podAnnotations := pod.GetAnnotations()
+	if cmCPUPolicyName == PolicyStatic && hint != nil {
+		if podAnnotations == nil {
+			pod.SetAnnotations(map[string]string{allowCPUQuotaDisabled: "true"})
+		} else if _, ok := podAnnotations[allowCPUQuotaDisabled]; !ok {
+			podAnnotations[allowCPUQuotaDisabled] = "true"
+		}
+	} else if podAnnotations == nil {
+		pod.SetAnnotations(map[string]string{allowCPUQuotaDisabled: "false"})
+	} else {
+		podAnnotations[allowCPUQuotaDisabled] = "false"
+	}
+	return hint
+}
+
+// CanDisablePodCPUQuota is used by runtime pod
+func CanDisablePodCPUQuota(pod *v1.Pod) bool {
```

Comment:
this seems to say the feature to enable or disable cfs quota is per pod.  i do not think that should be a granular pod setting.  if the desire is to not set cfs quota on containers that have an exclusive cpu set, we should make that decision globally without needing this annotation.

Reply:
This bring up a question: is it safe to globally disable cpu cfs quota?

Reply:
> this seems to say the feature to enable or disable cfs quota is per pod. i do not think that should be a granular pod setting. if the desire is to not set cfs quota on containers that have an exclusive cpu set, we should make that decision globally without needing this annotation.

Just got what you meant. Ok let me look into this.

Reply:
@derekwaynecarr Derek, circle back on this PR (sorry I was distracted by other tasks in the past weeks). I didn't figure out a good way to tell which pods/containers have an exclusive cpu set in order to do what you suggested, though I agree with the idea. Suggestions?